contrib/thrifty/src/python/pants/contrib/thrifty/BUILD

```diff
@@ -0,0 +1 @@
+python_library(sources=globs('*.py'))
```

Comment:
I think that you will want a `contrib_plugin` target here; example: https://github.com/pantsbuild/pants/blob/d30cca1e0ecb9cc0e1b7e2cd0ff6e7e077e62a52/contrib/scrooge/src/python/pants/contrib/scrooge/BUILD